pin_n3h

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env node
+
+/* Helper script for pinning downloadable version of n3h binaries.
+ * Bakes the sha256 hashes into the code for security.
+ *
+ * Run with the target n3h release name (tag name) e.g.
+ * `./pin_n3h adhoc-2019-03-19`
```

Comment:
@Connoropolous

> There is so far no documentation added to the repo itself on how to utilize the added script to update the pinned version.

Here's the documentation. Happy to put it somewhere else, too, if you have a suggestion where.

Reply:
Ok cool. I will have a quick think about where to put it, but in addition, I have the question:

who should update this version, and when? do you believe?

Reply:
Anyone iterating on n3h... I think the flow should be:

- bump n3h version in n3h feature branch
- manually bump pinned version in holochain-rust feature branch
- iterate
- code review
- code merged to n3h master
- adhoc n3h release / tag
- run n3h_pin script in holochain-rust feature branch
- code merged to holochain-rust develop

Then we no longer need to cut separate "releases" for n3h whenever holochain-rust releases are cut.

Reply:
thanks for laying this out. I think I'm picking up what you're putting down